src/components/Input/index.tsx

```diff
@@ -0,0 +1,7 @@
+import styles from "./Input.module.scss";
+
+const Input = () => {
+  return <input className={styles["input"]} />;
```

Comment:
refactor it to remove `return`

Reply:
https://github.com/igorizviekov/light-up/pull/7/commits/794e3f1b32505b24dd56d4254d2c216adbb5ee4d